qcfractal/interface/collections/dataset.py

```diff
@@ -35,11 +35,14 @@ class MoleculeEntry(ProtoModel):
 
 class ContributedValues(ProtoModel):
     name: str
-    doi: Optional[str] = None
+    citations: Optional[List[Citation]] = None
     theory_level: Union[str, Dict[str, str]]
     theory_level_details: Optional[Union[str, Dict[str, Optional[str]]]] = None
     comments: Optional[str] = None
-    values: Dict[str, Any]
+    values: Optional[Any] = None
```

Comment:
```
values: Union[Array, List[Any]]`
index: Array[str]
```

@mattwelborn ?

Reply:
Is `Array` the numpy array type?

Reply:
Yes.

Reply:
That's fine with me. We do need to update the contributed values functions, btw.

Reply:
Actually, I'm a little surprised that the tests pass...

Reply:
Also, what about `Optional`?